ts/src/time-profiler.ts

```diff
@@ -141,5 +168,9 @@ export function isStarted() {
   return !!gProfiler;
 }
 
+export function v8ProfilerStuckEventoopDetected() {
```

Comment:
```suggestion
export function v8ProfilerStuckEventLoopDetected() {
```

Reply:
Woops, thanks for the review !
I added some tests :)